terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/pagination/searchPathTmplAndQueryTmplAndCriteriaQuery.jsp

```diff
@@ -2,56 +2,57 @@
 <h1 class="center-content">Person Search Form</h1>
 
 <form class="center-content">
-    <spring:nestedPath path="personSearchForm">
-        <form:input path="name" />
-        <form:button id="searchButton">Search</form:button>
-    </spring:nestedPath>
+	<spring:nestedPath path="personSearchForm">

```

Comment:
@ikeyat 
 Sorry, It's difference that may occur due to I had to rebase too much.
 I Canceled the wrong rebase, and re-push only the necessary commits.
 This difference does not occur.